src/App.tsx

```diff
@@ -42,11 +46,17 @@ export const App = (): JSX.Element => {
   } else {
     return (
       <div className='pokemon-wrapper'>
+        Choosed pokemons:{' '}
+        {selectedPokemons.map((pokemon) => {
+          return <div>{pokemon}</div>;
+        })}
```

Comment:
Is this should to be here?

Reply:
This feature is for checking of working of logic. It will be removed.